.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -0,0 +1,8 @@
+
+[//]: # (This should be discussed in the daily meeting)
```

Comment:
This is just a simple hack to add some comments to the PR creator but without including them on the rendered PR.

For instance, this shows that the comment and surrounding blank lines will be ignored.

<img width="520" alt="Captura de tela 2023-11-03 124334" src="https://github.com/HathorNetwork/hathor-wallet-headless/assets/15909384/409ece18-7f51-43e4-8666-516154155ade">


<img width="471" alt="Captura de tela 2023-11-03 124342" src="https://github.com/HathorNetwork/hathor-wallet-headless/assets/15909384/82c8d420-c579-4486-a0f3-4894155f2a02">